### ISSUE TITLE
♿Combobox AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1347]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
- Each form input must have an associated label
<img width="693" alt="screen shot 2018-10-25 at 5 54 17 pm" src="https://user-images.githubusercontent.com/40970507/47535009-218af080-d87f-11e8-98fb-a7e7ca637b12.png">

- Both inputs have the same id.
<img width="677" alt="screen shot 2018-10-25 at 5 56 20 pm" src="https://user-images.githubusercontent.com/40970507/47535054-58610680-d87f-11e8-8540-4719e5ddca83.png">

- [The input element with WAI-ARIA property aria-activedescendant set to downshift-1-item-2 does not have tabindex attribute set to 0 or -1. See rule help for remediation guidance.
](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1084.html)<img width="679" alt="screen shot 2018-10-25 at 5 58 34 pm" src="https://user-images.githubusercontent.com/40970507/47535096-a2e28300-d87f-11e8-88d3-8e523fc14c0a.png">

- [The WAI-ARIA role(s) and/or attribute(s) combobox, aria-autocomplete, aria-expanded are not valid for the element input. See rule help for guidance.
](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1146.html)
The WAI-ARIA role(s) and/or attribute(s) combobox, aria-autocomplete, aria-expanded, aria-activedescendant are not valid for the element input. See rule help for guidance.
<img width="679" alt="screen shot 2018-10-25 at 6 00 23 pm" src="https://user-images.githubusercontent.com/40970507/47535139-dd4c2000-d87f-11e8-80e4-d988120edd7f.png">

- [Active descendant of composite widget with DOM focus must be valid.
](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1186.html)<img width="678" alt="screen shot 2018-10-25 at 6 03 55 pm" src="https://user-images.githubusercontent.com/40970507/47535280-5fd4df80-d880-11e8-8271-b7e3c149b3cb.png">

- **The input element with WAI-ARIA role combobox might not be keyboard accessible. See rule help for remediation guidance.**
> Tested and items inside box cannot be reached via keyboard

- [An element with WAI-ARIA role combobox does not contain or own at least one child element with each of the following WAI-ARIA roles: listbox, textbox, tree, grid, dialog. See rule help for remediation guidance.
](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1152.html)

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset
